src/sentry/api/endpoints/project_rule_async_task.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+from django.conf import settings
+
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint, ProjectSettingPermission
+from sentry.api.serializers import serialize
+from sentry.models import Rule, RuleStatus
+
+from sentry.utils import json
+from sentry.utils.redis import redis_clusters
+
+
+def _get_value_from_redis(task_uuid):
+    cluster_key = getattr(settings, "SENTRY_RULE_TASK_REDIS_CLUSTER", "default")
+    client = redis_clusters.get(cluster_key)
+    key = u"slack-channel-task:1:{}".format(task_uuid)
+    value = client.get(key)
+    return json.loads(value)
+
+
+class ProjectRuleTaskEndpoint(ProjectEndpoint):
+    permission_classes = [ProjectSettingPermission]
```

Comment:
@MeredithAnya I think you can omit this because the base class already has this set